go-controller/pkg/ovn/external_ids_syncer/acl/acl_sync.go

```diff
@@ -363,7 +363,8 @@ func (syncer *aclSyncer) updateStaleDefaultDenyNetpolACLs(legacyACLs []*nbdb.ACL
 	deleteOps []libovsdb.Operation, err error) {
 	for _, acl := range legacyACLs {
 		// sync default Deny policies
-		if acl.ExternalIDs[defaultDenyPolicyTypeACLExtIdKey] == "" {
+		if acl.ExternalIDs[defaultDenyPolicyTypeACLExtIdKey] == "" || acl.Priority == types.DefaultMcastDenyPriority ||
+			acl.Priority == types.DefaultMcastAllowPriority {
```

Comment:
I am confused here. Isn't the intention not to match multicast policies and make the filter more specific as stated in the PR message? Isn't OR'ing making the filter less specific? 

Did you mean to say multicast ACLs also use `defaultDenyPolicyTypeACLExtIdKey` external Id but with a different value?

Maybe add a comment explaining what we intend to match?

Reply:
Added a comment, hopefully it helps. 
To clarify, this `if` filters out not default deny acls, it continues with `// not default deny policy` comment. So with this change multicast acls will be matched by this `if` and filtered out